### PR TITLE
Hacky change to be able to only provide speedtest on localhost or lis…

### PR DESCRIPTION
**Pull request by twhuhn**
…ten to other addresses

### CODE REVIEW

File: prometheus_speedtest/prometheus_speedtest.py [hunk 1]
```diff
@@ -21,6 +21,13 @@
 PARSER = argparse.ArgumentParser(
     description='Instrument speedtest.net speedtests from Prometheus.',
     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+PARSER.add_argument(
+    '-a',
+    '--address',
+    metavar='address',
+    default='',
```
Comment (jeanralphaviles) [inline prometheus_speedtest/prometheus_speedtest.py]: Maybe we should default this to '0.0.0.0' so its obvious that we listen on all addresses by default.

File: prometheus_speedtest/prometheus_speedtest.py [hunk 2]
```diff
@@ -21,6 +21,13 @@
 PARSER = argparse.ArgumentParser(
     description='Instrument speedtest.net speedtests from Prometheus.',
     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+PARSER.add_argument(
```
Comment (jeanralphaviles) [inline prometheus_speedtest/prometheus_speedtest.py:24]: Could you put the new output of **prometheus_speedtest --help** in the usage section of [README.md](https://github.com/jeanralphaviles/prometheus_speedtest/blob/050b234/README.md#Usage)?

**Review by jeanralphaviles — CHANGES_REQUESTED** [review]

**Comment by jeanralphaviles** [comment]
> Hacky change....
This looks great to me, its how I would implement it. Thank you for the PR.

**Comment by twhuhn** [comment]
There you go ;)

**Review by twhuhn — COMMENTED** [review]
And it's a new feature, hence the 0.7.0

File: README.md [hunk 3]
```diff
@@ -43,6 +43,8 @@ Instrument speedtest.net speedtests from Prometheus.
 
 optional arguments:
   -h, --help            show this help message and exit
+  -a address, --address address
```
Comment (jeanralphaviles) [inline README.md:46]: You missed [-a address] after [-h] in the "usage:" line

usage: prometheus_speedtest.py [-h] [-a address] [-p port] [-v]

**Review by jeanralphaviles — CHANGES_REQUESTED** [review]
Just one nit, and then it'll be good to go.

**Comment by jeanralphaviles** [comment]
v0.7.0 has been pushed to PyPi: <https://pypi.org/project/prometheus-speedtest> and DockerHub: <https://hub.docker.com/r/jraviles/prometheus_speedtest>

Thanks for the PR.

**Pull request closed without merging**